packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -421,35 +421,15 @@ class ScrollableBase extends Component {
 
 		// Should skip scroll by page when focusedItem is paging control button of Scrollbar
 		if (focusedItem && childRefCurrent.containerRef.current.contains(focusedItem)) {
-			const
-				// VirtualList and Scroller have a spotlightId on containerRef
-				spotlightId = containerRef.current.dataset.spotlightId,
-				rDirection = reverseDirections[direction],
-				viewportBounds = containerRef.current.getBoundingClientRect(),
-				focusedItemBounds = focusedItem.getBoundingClientRect(),
-				endPoint = {
-					x: focusedItemBounds.left + focusedItemBounds.width / 2,
-					y: viewportBounds.top + ((direction === 'up') ? focusedItemBounds.height / 2 - 1 : viewportBounds.height - focusedItemBounds.height / 2 + 1)
-				};
-			let next = null;
-
-			/* 1. Find spottable item in viewport */
-			next = getTargetByDirectionFromPosition(rDirection, endPoint, spotlightId);
-
-			if (next !== focusedItem) {
-				Spotlight.focus(next);
-			/* 2. Find spottable item out of viewport */
-			// For Scroller
-			} else if (this.childRef.current.scrollToNextPage) {
-				next = this.childRef.current.scrollToNextPage({direction, focusedItem, reverseDirection: rDirection, spotlightId});
+			const spotlightId = containerRef.current.dataset.spotlightId;
+
+			if (this.childRef.current.scrollByPage) {
+				const next = this.childRef.current.scrollByPage({direction, focusedItem, reverseDirection: reverseDirections[direction], spotlightId});
```

Comment:
`spotlightId` is used just once. We don't need to define the const variable.

Reply:
I'll fix it.